package.json

```diff
@@ -40,14 +40,15 @@
     "@types/node": "^12.0.4",
     "@types/node-fetch": "^2.3.4",
     "concurrently": "^4.1.0",
-    "molstar": "^0.2.2",
+    "molstar": "../molstar",
```

Comment:
Don't commit the local path. I will accept the molstar PR #17 and do a new release to reference

Reply:
Reverted to original definition